### ISSUE TITLE
"standard vendor file"?

### ISSUE THREAD

**Issue by ahmetalpbalkan** [kardianos/govendor#249]
It says:

> 
![image](https://cloud.githubusercontent.com/assets/159209/20284893/eb664ec2-aa73-11e6-9c72-e11bff67b50c.png)

what is the "standard vendor file" though?

**Comment by tejasmanohar** [comment]
vendor/vendor.json

**Comment by ahmetalpbalkan** [comment]
@tejasmanohar yeah but what makes it standard?

**Comment by tejasmanohar** [comment]
Yeah not sure that the file is a standard, just the folder I think

**Comment by DennyLoko** [comment]
@kardianos made a spec of how the vendor file should be. Is this one: https://github.com/kardianos/vendor-spec

**Comment by robvdl** [comment]
I think this is misleading too, if you make your own spec, that doesn't make it a "standard vendor file".  There is no such thing as a standard vendor file yet as far as I am aware.
